### DISCUSSION TITLE
Typescript error on using @cypress/grep

### DISCUSSION THREAD

**Discussion by poponuts** [cypress-io/cypress#25108] [Questions and Help]
does anyone know in Typescript how I can remove this error? I'm using @cypress/grep https://github.com/cypress-io/cypress/tree/develop/npm/grep 
```Argument of type '{ tags: string; }' is not assignable to parameter of type 'SuiteConfigOverrides'.
  Object literal may only specify known properties, and 'tags' does not exist in type 'SuiteConfigOverrides'.```

**Comment by portyu9** [comment]
Hi @poponuts 

This was an `@cypress/grep` typings problem around the Cypress 11 migration, especially for tags applied to `describe()` blocks.

The preferred fix is to **upgrade `@cypress/grep`** rather than manually redefining Cypress's `SuiteConfigOverrides`.

With a current version, make sure its types are visible to TypeScript:

```json
{
  "compilerOptions": {
    "types": [
      "cypress",
      "@cypress/grep"
    ]
  }
}
```

Alternatively, add:

```ts
/// <reference types="@cypress/grep" />
```

to the relevant Cypress TypeScript declaration/spec context.

The exact error:

```text
Object literal may only specify known properties,
and 'tags' does not exist in type 'SuiteConfigOverrides'
```

was reported against the old `@cypress/grep` typings and subsequently addressed upstream.

So I would not use `// @ts-ignore` as the permanent solution and would not add a competing local `SuiteConfigOverrides` definition unless you are forced to remain on that old package version.

If you are maintaining the original Cypress 11-era project, update to the newest `@cypress/grep` release compatible with that Cypress version. For a current Cypress project, use the current `@cypress/grep` package and its shipped types.

Please mark this answer as accepted if it helped, thank you.
